expense-splitter/src/components/expense/CreateExpense.jsx

```diff
@@ -320,6 +337,7 @@ export default function CreateExpense() {
 
           <select
             className="h-16"
+            disabled={modal.id ? true : false} //created within the group detail page
             name="group"
```

Comment:
Can you match the disabled group behavior found in edit expense where it utilizes cursor-not-allowed?

Reply:
Do we actually want to disable the drop down or just have it pre-selected?

Reply:
This is true, we came in hot and almost burnt the place down! 🔥
It shouldn't be disabled.